### DISCUSSION TITLE
add assets to multiple albums at once

### DISCUSSION THREAD

**Discussion by MD77MD** [immich-app/immich#14216] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

currently we can only add an asset to multiple albums one at a time... this takes time especially if your adding many assets... because we have to do the hunt and reselect  all the assets i want to add for every album i want to add them to. 

my solution is in two parts:
**1. the obvious solution** 
add the ability to add assets to multiple albums at once
![SmartSelect_20241119_001017_Vivaldi](https://github.com/user-attachments/assets/c3997657-b8f7-4aa9-9b5d-d8bae11ca18e)

**2. a great tool to save time**
 add an option in settings to keep the selection of assets after an operating (tagging, adding to album) is done. 

for example i select some assets and added a tag "friends".. but then i want to share them with my friends... instead of redoing the selection hunt, this feature would mean all those assets i just tagged are still selected and i can just add them to the album i want. i can even modify the selected assets (add or exclude) before i share them to the that album. clicking anywhere else removes the selection. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2686

**Comment by MD77MD** [comment]
my original post was about the adding assets to multiple albums ... the retaining selection was just added which is supposed to be put in different request (dorry)... so my request is actually not a duplicate 

any i have corrected this in a new post
